scripts-local/test-utils/run_any_cli_command.sh

```diff
@@ -4,20 +4,28 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 source ${SCRIPT_DIR}/../account_vars.sh
 
 
-# $GAIA_CMD tx ibc-transfer transfer transfer channel-0 stride1uk4ze0x4nvh4fk0xm4jdud58eqn4yxhrt52vv7 100000uatom --from gval1 --chain-id GAIA -y --keyring-backend test
```

Comment:
should we rm this file? seems like scratch

Reply:
it's useful imo and low overhead to keeping it